oe_theme.theme

```diff
@@ -210,153 +210,352 @@ function oe_theme_preprocess_paragraph__oe_links_block(array &$variables): void
  * Implements hook_preprocess_oe_corporate_blocks_ec_footer().
  */
 function oe_theme_preprocess_oe_corporate_blocks_ec_footer(array &$variables): void {
+  $variables['processed'] = [];
   $corporate_footer = $variables['corporate_footer'];
-
-  // Create the section for 'About EC'.
-  $about_ec = $corporate_footer['about_ec'];
-  $sections_about_ec = [
-    'title' => $about_ec['title'],
-    'links' => [],
+  $site_name = $corporate_footer['site_name'];
+  $site_name_link = [
```

Comment:
This whole section is very confusing, we have a site_name_link but also a site_name and a site_path :( Maybe we could rename site_name_link to something link: ec_site_link or something